command/cmd_copy.go

```diff
@@ -532,7 +529,7 @@ func prepareCopyDestination(
 
 	// Absolute <src> path is given. Use given <dst> and local copy operation
 	// will create missing directories if <dst> has one.
-	if !origSrc.HasGlob() {
+	if !srcurl.HasGlob() {
```

Comment:
i think we should explicitly call srcurl.Origin().HasGlob(). `srcurl` itself is an absolute url, not a glob.